packages/opentelemetry-resources/src/platform/node/detectors/EnvDetector.ts

```diff
@@ -87,10 +87,7 @@ class EnvDetector implements Detector {
       let [key, value] = keyValuePair;
       // Leading and trailing whitespaces are trimmed.
       key = key.trim();
-      value = value
-        .trim()
-        .split('^"|"$')
-        .join('');
+      value = value.trim().split('^"|"$').join('');
```

Comment:
What rule triggered this rewrite?

Reply:
I believe that there is no rewrite, I believe that it's just "prettier" that makes one line.